base_layer/core/src/transactions/transaction_components/output_features_version.rs

```diff
@@ -17,7 +17,6 @@ use crate::consensus::{ConsensusDecoding, ConsensusEncoding, ConsensusEncodingSi
 pub enum OutputFeaturesVersion {
     V0 = 0,
     V1 = 1,
-    V2 = 2,
 }
```

Comment:
I think we should only have one version here, i.e. the latest

Reply:
Its only for unit testing
And later usage